community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -148,12 +148,6 @@ public NeoStore newNeoStore( boolean allowCreateEmpty )
             return createNeoStore();
         }
 
-        CountsTracker countsTracker = null;
-        if ( storeExists && countsStoreExists() )
-        {
-            countsTracker = newCountsStore();
-        }
-
```

Comment:
This code was needed to defer the counts store rebuilding after recover. Indeed if you have the other stores and the counts store is missing you cannot open it.